web/tests/compose_validate.test.js

```diff
@@ -817,3 +825,59 @@ test_ui("test warn_if_topic_resolved", ({override, mock_template}) => {
     compose_validate.warn_if_topic_resolved(false);
     assert.ok(!error_shown);
 });
+
+test_ui("test_warn_if_guest_in_dm_recipient", ({mock_template}) => {
+    let is_active = false;
+
+    mock_template("compose_banner/guest_in_dm_recipient_warning.hbs", false, (data) => {
+        assert.equal(data.classname, compose_banner.CLASSNAMES.guest_in_direct_message_recipient);
+        assert.equal(
+            data.banner_text,
+            $t({defaultMessage: "Guest is a guest in this organization."}),
```

Comment:
Let's also add a test for the plural case of guests.

Reply:
Tried doing it but there is an `input_pill` stub error when adding multiple recipients. The multiple guest filtering and naming test is being covered in `people.test.ts` as well so should I still go for it?

Reply:
For this, can you start a topic on CZO, it's hard to know what error you are getting without logs

Reply:
Any updates here?

Reply:
Yes fixed the error by overriding `ui_util.place_caret_at_end` and changed the second case to use 2 emails.